sources/packages/backend/workflow/test/2023-2024/parttime-assessment/parttime-assessment-2023-2024-eligibility-SBSD.e2e-spec.ts

```diff
@@ -0,0 +1,73 @@
+import { PROGRAM_YEAR } from "../constants/program-year.constants";
+import {
+  createFakeConsolidatedPartTimeData,
+  executePartTimeAssessmentForProgramYear,
+} from "../../test-utils";
+import { YesNoOptions } from "@sims/test-utils";
+import { InstitutionTypes } from "../../models";
+
+describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-eligibility-SBSD.`, () => {
+  it(
+    `Should determine SBSD as eligible when institution type is ${InstitutionTypes.BCPrivate}, ` +
+      "total assessment need is greater than 0 and application PD/PPD status is 'yes' for single student.",
+    async () => {
+      // Arrange
+      const assessmentConsolidatedData =
+        createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+      assessmentConsolidatedData.institutionType = InstitutionTypes.BCPrivate;
+      assessmentConsolidatedData.studentDataApplicationPDPPDStatus = "yes";
+
+      // Act
+      const calculatedAssessment =
+        await executePartTimeAssessmentForProgramYear(
+          PROGRAM_YEAR,
+          assessmentConsolidatedData,
+        );
+
+      // Assert
+      expect(calculatedAssessment.variables.awardEligibilitySBSD).toBe(true);
+      expect(
+        calculatedAssessment.variables.finalProvincialAwardNetSBSDAmount,
+      ).toBeGreaterThan(0);
+    },
+  );
+
+  it(`Should determine SBSD as not eligible when institution type is ${InstitutionTypes.International} for a married student.`, async () => {
+    // Arrange
+    const assessmentConsolidatedData =
+      createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+    assessmentConsolidatedData.studentDataRelationshipStatus = "married";
+    assessmentConsolidatedData.studentDataIsYourSpouseACanadianCitizen =
```

Comment:
Does it matter whether the student is married with a Canadian citizen or not?

Reply:
no @andrepestana-aot, it is just to cover different relationship status and studentDataDependantstatus scenario, I don't want to do all TC with a single student, that's its, these status doesn't have any effect, that was the purpose

Reply:
If that's the case I think the relationship shouldn't mentioned because it sounds like you're testing the relationship status.

Reply:
removed from all TCs